logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/message/MessageSender.kt

```diff
@@ -207,6 +238,29 @@ internal class MessageSenderImpl internal constructor(
             }
     }
 
+    private suspend fun attemptToBroadcastWithProteus(
+        message: BroadcastMessage,
+        target: BroadcastMessageTarget
+    ): Either<CoreFailure, String> {
+        return userRepository.getAllRecipients().flatMap { (teamRecipients, otherRecipients) ->
```

Comment:
according to [swagger](https://staging-nginz-https.zinfra.io/api/swagger-ui/#/default/post_broadcast_proteus_messages) 
```
report_only: Takes a list of qualified UserIDs. If any clients of the listed users are missing, the message is not sent. The missing clients are reported in the response.
```
meaning as long as we have the list of user IDs we can rely on the message sender error handling to add any missing clients and there is no need to fetch the full list of clients remotely

Reply:
`getAllRecipients()` gets clients from the DB, not remotely. 
So the error scenario would be: 
1. we get all "known clients" 
2. try sending a message to them 
3. API returns the list of the missed clients 
4. we fetch all missed clients and save them in DB
5. repeat from #1

Reply:
My bad, I thought it fetches the list of clients remotely.